### PR TITLE
[BML-2] Async prepare bills and summaries as datasets

### PR DESCRIPTION
**Pull request by BorodaUA**
Hi 👋 team, please review changes that I made in scope of following task(s):

- [BML-2] Prepare bills and summaries as datasets

Here what is done:

- Initial project setup (poetry, documentation)
- Added async python scripts to create `.jsonl` file dataset

Example execution time:
`Program exection time (hh:mm:ss.ms) 0:05:26.636805`

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "billml"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```
Comment (alexbojko) [inline pyproject.toml]: authors

File: utils/datasets.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+
+
+async def get_bill_data(bill_folder: str, num_bill_sections: int):
+    document_folders = get_bills_folders_with_document_file(bill_folder)
+    if not document_folders:
+        return
+    latest_document_filepath = get_latest_document_filepath(document_folders)
+    if not await check_bill_num_of_sections(num_bill_sections, latest_document_filepath):
+        _log.debug(
+            f'Bill with filepath: {latest_document_filepath} have more than: {num_bill_sections} sections.'
```
Comment (alexbojko) [inline utils/datasets.py]: bill `has`

File: utils/datasets.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        )
+        return
+    bill_sections = await get_bill_sections(latest_document_filepath)
+    bill_text = ' '.join([section_text['text'] for section_text in bill_sections])
```
Comment (alexbojko) [inline utils/datasets.py:92]: I would avoid creating variable `bill_text` which is used only once in a code

File: utils/datasets.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+    document_folders = get_bills_folders_with_document_file(bill_folder)
+    if not document_folders:
+        return
+    latest_document_filepath = get_latest_document_filepath(document_folders)
+    if not await check_bill_num_of_sections(num_bill_sections, latest_document_filepath):
+        _log.debug(
+            f'Bill with filepath: {latest_document_filepath} have more than: {num_bill_sections} sections.'
+        )
+        return
+    bill_sections = await get_bill_sections(latest_document_filepath)
```
Comment (alexbojko) [inline utils/datasets.py:91]: same as next comment

**Review by alexbojko — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (BorodaUA) [reply pyproject.toml]: Authors info added in the new commit.

File: utils/datasets.py (hunk 2 above)
Comment (BorodaUA) [reply utils/datasets.py]: This typo fixed in the new commit.

File: utils/datasets.py (hunk 3 above)
Comment (BorodaUA) [reply utils/datasets.py:92]: Turns out we do use this variable more than once, this part left unchanged in the new commit.

File: utils/datasets.py (hunk 4 above)
Comment (BorodaUA) [reply utils/datasets.py:91]: Typos fixed in the new commit.

**Pull request merged**